### PR TITLE
multiple changes

### PR DESCRIPTION
**Pull request by JesseBroxterman**

### CODE REVIEW

File: SolidWaste2/SW.BLL/Services/WorkOrderService.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ public async Task<WorkOrder> GetById(int id)
             .Where(e => e.WorkOrderId == id)
             .AsNoTracking()
             .FirstOrDefaultAsync();
+        //shouldn't this be Single or Default?
```
Comment (jessbrox809) [inline SolidWaste2/SW.BLL/Services/WorkOrderService.cs]: Remove comments, this should be Single.

File: SolidWaste2/SW.InternalWeb/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -36,7 +36,7 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
-    <title>@ViewData["Title"] - Solidwaste Billing</title>
+    <title>@ViewData["Title"] - Shawnee County Solidwaste Billing</title>
```
Comment (jessbrox809) [inline SolidWaste2/SW.InternalWeb/Views/Shared/_Layout.cshtml:39]: Did they ask for this to change?

**Review by jessbrox809 — APPROVED** [review]

File: SolidWaste2/SW.BLL/Services/WorkOrderService.cs (hunk 1 above)
Comment (JesseBroxtermanGmail) [reply SolidWaste2/SW.BLL/Services/WorkOrderService.cs]: Yes, that could cause a bug.

**Review by JesseBroxtermanGmail — CHANGES_REQUESTED** [review]

File: SolidWaste2/SW.BLL/Services/WorkOrderService.cs (hunk 1 above)
Comment (JesseBroxterman) [reply SolidWaste2/SW.BLL/Services/WorkOrderService.cs]: this has been fixed

**Review by JesseBroxtermanGmail — APPROVED** [review]

**Pull request merged**